int8 device
int8 mfc
int8 percent_capacity
---
string status